src/items.cpp

```diff
@@ -1273,93 +1259,92 @@ bool dropItem(Item* item, int player, bool notifyMessage)
 		}
 		return false;
 	}
-	else
```

Comment:
Personal taste - prefer the else {} as it indicates the below code is strictly for singleplayer/server

Reply:
Fixed in ba4ea3fb